app/assets/stylesheets/_global.sass

```diff
@@ -0,0 +1,16 @@
+// Shared Styles for the entire webapp
+
+@import "normalize-rails"
+@import "bourbon"
+
+/*!
+ * Author:       The Frontier Group
+ * Author URI:   http://www.thefrontiergroup.com.au/
+ */
```

Comment:
It's called a banner and I'd like that to be on top of all our CSS. Every library/plugin these days has one and it's an easy and "nice" way to credit us unobtrusively.